### ISSUE TITLE
Error when removing control from FormArray

### ISSUE THREAD

**Issue by jackcmac** [ng-turkey/ngx-validate#63]
Hello! I have a form in which sections can be added or removed by the user by using a `FormArray` like this:

```
private buildForm() {
  const dynamicSections = this.fb.array([this.createSection()]);
  this.form = this.fb.group({ dynamicSections });
}

public get getSections(): FormArray {
  return this.form.get('dynamicSections') as FormArray;
}

public addSection(): void {
  this.getSections.push(this.createSection());
}

private createSection(): FormGroup {
  return this.fb.group({
    prop1: [null],
    prop2: [null],
  });
}

public removeSection(index: number): void {
  this.getSections.removeAt(index);
}
```

However, when a section is removed, I am getting the following error:

```
ERROR TypeError: Cannot read property 'errors' of null
    at MapSubscriber.eval [as project] (validation.directive.ts:137)
    at MapSubscriber._next (map.ts:84)
    at MapSubscriber.Subscriber.next (Subscriber.ts:99)
    at FilterSubscriber._next (filter.ts:101)
    at FilterSubscriber.Subscriber.next (Subscriber.ts:99)
    at MergeMapSubscriber.notifyNext (mergeMap.ts:162)
    at SimpleInnerSubscriber._next (innerSubscribe.ts:30)
    at SimpleInnerSubscriber.Subscriber.next (Subscriber.ts:99)
    at MapToSubscriber._next (mapTo.ts:70)
    at MapToSubscriber.Subscriber.next (Subscriber.ts:99)
```

**To Reproduce**
1. Here is a StackBlitz of the issue: https://stackblitz.com/edit/ngx-validate-n8j7ib?devtoolsheight=33&file=src/app/components/app.component.ts
2. Notice the console error after clicking the `Remove` button

Let me know if you have any suggestions or need any other information.

Thanks!

**Comment by Dovernh** [comment]
I am also experiencing this same issue.  The item is removed, but error messages are logged in the browser console. 
![image](https://user-images.githubusercontent.com/1800249/112362326-b1c8fc00-8caa-11eb-8e94-316f79c1ba6d.png)

**Comment by armanozak** [comment]
Hi @jackcmac  and @Dovernh,

Thank you for reporting the error. We have a PR that fixes it. There will be a new release (v0.1.1) right after the PR.

Have a nice day.

**Comment by dhoorek** [comment]
Hi, any update on this?

**Comment by armanozak** [comment]
Hi. It should be fixed with v0.1.1 release.
